### PR TITLE
'changed_contribution_steps'

### PR DESCRIPTION
**Pull request by catrawalkar**
### Description

Just getting started, so just added few documentation lines.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -92,7 +92,9 @@ render(<App />, document.getElementById('root'));
 
 ### For development
 
-1. At the project root, run `yarn install`
+1. Fork this repository using by clicking _Fork_ button at the top
+1. Clone your own version on repository by using `git clone https://github.com/<username>/reactfire.git` (replace the repo url with your own version's url)
```
Comment (jhuleatt) [inline README.md]: I think we can simplify the first two down to:

```md
1. [Clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repository (or a [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo#propose-changes-to-someone-elses-project))
```

Which will look like this:

1. [Clone](https://help.github.com/en/github/creating-cloning-and-archiving-repositories/cloning-a-repository) this repository (or clone a [fork](https://help.github.com/en/github/getting-started-with-github/fork-a-repo#propose-changes-to-someone-elses-project))

File: README.md [hunk 2]
```diff
@@ -92,7 +92,9 @@ render(<App />, document.getElementById('root'));
...
+1. At the project root, install all modules by running `yarn install` or `npm install`
```
Comment (jhuleatt) [inline README.md]: Please remove `npm install`. This project uses a [yarn workspace](https://yarnpkg.com/lang/en/docs/workspaces/) so `npm install` likely won't work.

**Comment by jhuleatt** [comment]
Thanks for the PR, @catrawalkar! I've added a few comments, but once you update those things I'll merge this.

**Comment by catrawalkar** [comment]
@jhuleatt  done :)

**Pull request merged**